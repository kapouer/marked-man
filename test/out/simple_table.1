SIMPLE_TABLE()                                                  SIMPLE_TABLE()

NNAAMMEE
       ssiimmppllee__ttaabbllee

       ┌──────────────────────────────────────────────────────────────────────┐
       │      Col1                      Col2                       Col3       │
       ├──────────────────────────────────────────────────────────────────────┤
       │      A                           B                           C       │
       ├──────────────────────────────────────────────────────────────────────┤
       │      D                           E                           F       │
       └──────────────────────────────────────────────────────────────────────┘

                                 January 1979                   SIMPLE_TABLE()
