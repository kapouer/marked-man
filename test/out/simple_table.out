[4mSIMPLE_TABLE[24m(1)             General Commands Manual            [4mSIMPLE_TABLE[24m(1)

[1mNAME[0m
       [1msimple_table[0m
       ┌──────────────────────────────────────────────────────────┐
       │ Col1                      Col2                      Col3 │
       ├──────────────────────────────────────────────────────────┤
       │ A                           B                          C │
       ├──────────────────────────────────────────────────────────┤
       │ D                           E                          F │
       └──────────────────────────────────────────────────────────┘

                                 January 1979                  [4mSIMPLE_TABLE[24m(1)
